src/content.ts

```diff
@@ -1218,7 +1220,7 @@ function getYouTubeVideoID(document: Document, url?: string): string | boolean {
     // pageType shortcut
     if (pageType === PageType.Channel) return getYouTubeVideoIDFromDocument()
     // clips should never skip, going from clip to full video has no indications.
-    if (url.includes("youtube.com/clip/")) return false;
+    if (url.includes("youtube.com/clip/")) return "clipNoID";
```

Comment:
What's the reason for making a string here?

Reply:
otherwise it'll be overwritten by "false"